### PR TITLE
Pull request to add classes

### PR DESCRIPTION
**Pull request by kimothokamau**
- Refactored the code to use different classes

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -1,72 +1,84 @@
...
+    }
+    return library;
+  }
+
+  static setBook(book) {
+    const books = Storage.getBooks();
+    books.push(book);
+    library = books;
+    localStorage.setItem('library', JSON.stringify(books));
+  }
```
Comment (de-scholar) [inline main.js]: - [ ] This method should be inside class Book

File: main.js [hunk 2]
```diff
@@ -1,72 +1,84 @@
...
-function getBooks() {
-  if (localStorage.getItem('library')) {
-    library = JSON.parse(localStorage.getItem('library'));
-  } else {
-    library = [];
+
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
```
Comment (de-scholar) [inline main.js:15]: - [ ] The class Book should have at least four methods: `constructor`, `addBook`, `removeBook`, `getBooks`

File: main.js [hunk 3]
```diff
@@ -1,72 +1,84 @@
...
+                        <button class ='remv-cls'>Remove</button>`;
+    bookContainer.appendChild(bookDiv);
+  }
+
+  static removeBook(element) {
+    const books = Storage.getBooks();
+    const indexBook = Array.prototype.indexOf.call(
+      bookContainer.childNodes,
+      element.parentElement,
+    );
```
Comment (de-scholar) [inline main.js]: - [ ] These methods should be inside class Book

**Review by de-scholar — CHANGES_REQUESTED** [review]
Hi @kimothokamau and @Jmagero , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._
- [ ] There is no need to make all methods static, try to make them class methods and that should work fine

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kimothokamau** [comment]
- Removed static methods
- Refactored class methods

**Review by sinansevgi — APPROVED** [review]
# Approved :heavy_check_mark: 

Hi @Jmagero and @kimothokamau , 

Great work with making all the changes requested by the previous TSE.
Your project is complete!  
Congratulations! 🎉

You can merge this pull request and continue with your journey :wave: 


Happy Hacking! :computer: 

-----

You can reply to this conversation if you have any questions or need elaboration. :wave: 
Sinan Sevgi


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**